web/components/bet/limit-slider.tsx

```diff
@@ -0,0 +1,143 @@
+import clsx from 'clsx'
+import {
+  CPMMBinaryContract,
+  CPMMMultiContract,
+  PseudoNumericContract,
+  StonkContract,
+} from 'common/contract'
+import Slider from 'rc-slider'
+import { useEffect, useRef, useState } from 'react'
+import { useWindowSize } from 'web/hooks/use-window-size'
+import { Col } from '../layout/col'
+import { Row } from '../layout/row'
+
+export function LimitSlider(props: {
+  lowLimitProb: number
+  setLowLimitProb: (prob: number) => void
+  highLimitProb: number
+  setHighLimitProb: (prob: number) => void
+  mobileView?: boolean
+}) {
+  const {
+    lowLimitProb,
+    setLowLimitProb,
+    highLimitProb,
+    setHighLimitProb,
+    mobileView,
+  } = props
+
+  const mark = (value: number) => (
+    <span className="text-ink-400 text-xs">
+      <div className={'sm:h-0.5'} />
+      {value}%
```

Comment:
Have you checked if this works with the Pseudo-numeric type? (The kind Austin used to use for his Taco Tuesday markets).

Instead of percents, you would show numbers chosen within the lower and upper bound.

I'd be ok shipping with percents too as long as it still places limit orders correctly for pseudo numeric.